tests/src/python/test_qgsserver_wms_getmap_size_project.py

```diff
@@ -45,7 +45,7 @@ def setUp(self):
         os.environ['QGIS_SERVER_WMS_MAX_HEIGHT'] = '6000'
         super(TestQgsServerWMSGetMapSizeProject, self).setUp()
         self.project = os.path.join(self.testdata_path, "test_project_with_size.qgs")
-        self.expected_too_big = self.strip_version_xmlns(b'<ServiceExceptionReport version="1.3.0" xmlns="http://www.opengis.net/ogc">\n <ServiceException code="InvalidParameterValue">The requested map size is too large</ServiceException>\n</ServiceExceptionReport>\n')
+        self.expected_too_big = self.strip_version_xmlns(b'<ServiceExceptionReport version="1.3.0" xmlns="http://www.opengis.net/ogc">\n <ServiceException code="Size error">The requested map size is too large</ServiceException>\n</ServiceExceptionReport>\n')
```

Comment:
InvalidParameterValue is the correct value, "Size error" is not in the specs, IIRC this was fixed in master with https://github.com/qgis/QGIS/pull/9646